SwiftWisdomTests/StandardLibrary/Numbers/UnsignedInteger+ExtensionsTests.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UnsignedInteger+ExtensionsTests.swift
+//  SwiftWisdom
+//
+//  Created by Tom O'Malley on 9/30/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import XCTest
+import SwiftWisdom
+
+class UnsignedIntegerTests: XCTestCase {
+
+    func testRandomInRange() {
+        let positives = 0...100
+        XCTAssert(positives.contains(random(inRange: positives)))
```

Comment:
just a note here - testing random functions are tricky, since even if it passes for you, it might not pass for me. But what the heck..if it fails for someone we will know something is up.